docs/docker/README.md

```diff
@@ -1,44 +1,64 @@
 # Minio Docker Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
 
-## Prerequisites
-Docker installed on your machine. Download the relevant installer from [here](https://www.docker.com/community-edition#/download).
+This quickstart guide describes how to quickly install and run a Minio Server Docker container. These are the steps you will follow:
```

Comment:
Remove familiar references (you, et. al.) as discussed

Reply:
@eco-minio This is done. Please see https://github.com/jamesneimanconsulting/minio/blob/patch-2/docs/docker/README.md.